aten/src/ATen/gen_backend_select_register.py

```diff
@@ -24,12 +24,27 @@
 GENERATED_COMMENT = CodeTemplate(
     "@" + "generated from ${filename}")
 
-FUNCTION_REGISTRATION = CodeTemplate("""\
+UNBOXEDONLY_FUNCTION_REGISTRATION = CodeTemplate("""\
   m.impl_UNBOXED("aten::${op_name_with_overload_name}", ${function_name});
 """)
 
+FUNCTION_REGISTRATION = CodeTemplate("""\
+  m.impl("aten::${op_name_with_overload_name}", c10::impl::hacky_wrapper_for_legacy_signatures(TORCH_FN(${function_name})));
+""")
+
 FUNCTION_DEFINITION = CodeTemplate("""\
 // ${schema_string}
+Tensor ${function_name}(${method_formals}) {
```

Comment:
Same suggestion as in `function_wrapper` to use different template variables rather than copy pasting the template